core/local/SyncDir.js

```diff
@@ -0,0 +1,32 @@
+/* @flow */
```

Comment:
Well, CommonJS says it should be [lowerCamelCase](http://wiki.commonjs.org/wiki/Modules/1.1#Module_Identifiers), but Node.js has underscores (which feels alien to almost everything else in js) or sometimes not like with the `path` module (forget about any `path` variable), then React has CamelCase even for modules, which looked nice to me except you may end up with the module name conflicting with flow type names... Module naming is a pain in js :confused: 

I agree that we should stick to a single convention, and in case we decide to change we should do it all at once (my bad).

How would you expect the module to be aliased?
`const sync_dir = require(...)`? Or `const syncDir = require(...)`?

Reply:
I was talking of the filename, not the module its-self.

For the module, I try to use lowerCamelCase when it's a bag of functions, and CamelCase when it's a class that can be instanciated. But, yeah, naming in JS is not very consistent between the projects/ecosystems.

Reply:
:ok_hand: Fixed